src/util/backend.ts

```diff
@@ -1,9 +1,6 @@
 import { BACKEND_URL } from './constants';
 
-export interface IErrorResponse {
-	code: number;
-	message: string;
-}
+export type UserType = 'mentor' | 'student';
```

Comment:
ideally this type should go into a global styles file because it is not confined to backend